mantle/sdk/distros.go

```diff
@@ -47,3 +47,26 @@ func TargetIgnitionVersion(build *cosa.Build) string {
 	// Most cosa builds should have an "ostree"
 	return TargetIgnitionVersionFromName(build.BuildArtifacts.Ostree.Path)
 }
+
+// TargetDistroFromName returns the distribution given
+// the path to an artifact (usually a disk image).
+func TargetDistroFromName(artifact string) string {
+	basename := filepath.Base(artifact)
+	if strings.HasPrefix(basename, "rhcos-") {
+		return "rhcos"
+	}
+	// For now, just assume fcos
+	return "fcos"
+}
+
+// TargetDistro returns the distribution of a cosa build
+func TargetDistro(build *cosa.Build) (string, error) {
+	switch build.Name {
+	case "rhcos":
+		return "rhcos", nil
+	case "fedora-coreos":
+		return "fcos", nil
+	default:
```

Comment:
Should this return `fcos` as well to match `TargetDistroFromName`

Reply:
I think it's fair to fail in the case that it's using build metadata and receives unexpected results.

Reply:
Probably though in the future we should support the equivalent of `ID_LIKE` from `/usr/lib/os-release` in cosa metadata to aid derivatives i.e.:

```
name: foobar-os
osid-like: fedora-coreos
```